springfox-swagger-common/src/main/java/springfox/documentation/swagger/annotations/Annotations.java

```diff
@@ -52,8 +53,16 @@ private Annotations() {
     return fromNullable(findAnnotation(annotated, ApiOperation.class));
   }
 
-  public static Optional<ApiResponses> findApiResponsesAnnotations(Method annotated) {
-    return fromNullable(findAnnotation(annotated, ApiResponses.class));
+  public static List<Optional<ApiResponses>> findApiResponsesAnnotations(AnnotatedElement annotated) {
```

Comment:
`List` is monadic in itself, so just a returning a list of ApiResponses would suffice. i.e. don't add it to the list of the `getAnnotation` method returns a null. makes sense? 

It'll make consumption in downstream processes easier, for e.g. we won't need a `if (apiResponses.isPresent()) {...}` block.